src/test/unit/DelegationUnit.t.sol

```diff
@@ -1,3236 +1,3236 @@
-// SPDX-License-Identifier: BUSL-1.1
-pragma solidity =0.8.12;
```

Comment:
Nit: think most of this file was not meant to be modified?

Can we potentially add tests to a separate `AVSDirectory.unit.t.sol` file?

Reply:
Done, moved old tests to AVSDirectoryUnit.t.sol and left commented out for now
https://github.com/Layr-Labs/eigenlayer-contracts/pull/403/commits/a2dafdc78d22a375f48e4919478c153d24722a62